packages/react-form/src/hooks/field/reducer.ts

```diff
@@ -103,7 +104,7 @@ export function makeFieldReducer<Value>({
           return {
             ...state,
             error: firstError,
-            allErrors: payload,
+            allErrors: allErrors,
```

Comment:
Since the key and the variable have the same name you can simply do this
```suggestion
            allErrors,
```